### PR TITLE
Add Depot to CI/CD tools

### PR DESCRIPTION
**Pull request by kylegalbraith**
Resources to read more about it:
https://depot.dev/docs#when-to-use-depot
https://depot.dev/docs/integrations/github-actions
https://depot.dev/#benchmarks

### CODE REVIEW

**Comment by agamm** [comment]
Looks great! adding.

File: README.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ The definition of developer-first for this repo is:
 * [Gitlab](https://gitlab.com/) - Open source CI and hosted solution with Github-like abilities.
 * [Gitpod](https://www.gitpod.io/) - Automated cloud dev environments. 
 * [Spacelift](https://spacelift.io/) - CI/CD platform for infrastructure-as-code, currently supporting Terraform, Pulumi, AWS CloudFormation, and Kubernetes.
+* [Depot](https://depot.dev) - Build Docker images fast, in the cloud.
```
Comment (agamm) [inline README.md:84]:
```suggestion
* [Depot](https://depot.dev) - Fast drop-in replacement to major docker images for your CI.
```

Comment (agamm) [reply README.md:84]: This feels more concrete, no?

**Review by agamm — CHANGES_REQUESTED** [review]
I changed the description a bit to be less vague. Is it correct?

File: README.md (hunk 1 above)
Comment (kylegalbraith) [reply README.md:84]: I think `Drop-in replacement to build docker images faster in CI` is more accurate and concrete. Thoughts?

Comment (agamm) [reply README.md:84]: Great, change that and let's merge?

Comment (kylegalbraith) [reply README.md]: Done ✅

**Review by agamm — APPROVED** [review]
⚡

**Pull request merged**